src/plugin/featureaction/featureactionmenu.js

```diff
@@ -19,7 +19,6 @@ plugin.im.action.feature.layerSetup = function() {
       eventType: plugin.im.action.feature.EventType.LAUNCH,
       tooltip: 'Perform actions on imported data matching a filter',
       icons: ['<i class="fa fa-fw ' + os.im.action.ICON + '"></i>'],
-      beforeRender: plugin.im.action.feature.visibleIfSupported,
```

Comment:
This will show the Feature Actions option for all layers, which is not desirable. We still want to filter out non-vector layers, and reference layers (drawing/search/etc).